### ISSUE TITLE
Edit in review comments 📖

### ISSUE THREAD

**Issue by KarelZe** [KarelZe/thesis#317]
## Highlights

 * Page 8 (Tick Test): ","

 * Page 9 (Trade Size Rule): "however,"

 * Page 10 (Hybrid Rules):
   > As put forth by Grauer et al. (2022, p. 18), basic or hybrid rules can be combined

 * Page 16 (Framing as a Supervised Learning Problem): "Each data instance"

 * Page 16 (Framing as a Supervised Learning Problem): "Each data instance"

 * Page 16 (Framing as a Supervised Learning Problem): "ach da"

 * Page 17 (Selection of Approaches (2 p)): "TODO"

 * Page 19 (Decision Tree):
   > contributes to the model's overall expected error. Both extremes are to be avoided.

 * Page 19 (Decision Tree): "m"

 * Page 28 (Positional Encoding):
   > Like token embeddings, positional embeddings can also be learned (cp. Devlin et al.

 * Page 28 (Positional Encoding): "."

 * Page 29 (Positional Encoding):
   > The reason for encoding with both the sine and cosine is more subtle, as either one would suffice for absolute embeddings. Vaswani et al. (2017, p. 6) hypothesize,

 * Page 30 (Attention Mechanism):
   > Similar to a soft dictionary lookup, the goal is to retrieve the values from tokens in the context for which the keys are similar to the query and return an aggregate estimate of the values weighted by the similarity of the keys and the query.

 * Page 30 (Attention Mechanism): "c"

 * Page 30 (Attention Mechanism): "c"

 * Page 30 (Attention Mechanism): "c"

 * Page 34 (Residual Connections and Layer Normalization): "Intuitively,"

 * Page 36 (FT-Transformer): "classification."

 * Page 37 (FT-Transformer): "learning on partially labelled data."

 * Page 41 (Gradient Boosted Trees With Self-Training):
   > At the same time, self-training puts a high emphasis on the correctness of the prob

 * Page 44 (Data and Data Preparation (6 p)): "The following chapter"

 * Page 44 (Data and Data Preparation (6 p)): "We also discuss"

 * Page 44 (Data and Data Preparation (6 p)): "The following chapter"

 * Page 44 (Data and Data Preparation (6 p)): "We also discuss"

 * Page 46 (Data Collection): "with multiple underlying symbols for the same root."

 * Page 47 (Data Collection):
   > distinguish by trader types, this dataset includes both customer and non-customer trades.

 * Page 48 (Data Preprocessing): "O"

 * Page 48 (Data Preprocessing): "O"

 * Page 48 (Data Preprocessing): "ur"

 * Page 48 (Data Preprocessing): "Our l"

 * Page 57 (Hyperparameter Tuning (2 p)):
   > The loss surface of the GBM trained on the feature set including option features

 * Page 61 (Evaluation Metric): "Intuitively, fro"

 * Page 64 (Feature Importance Measure):
   > In the tabular domain, various approaches have been investigated in the literature to

 * Page 66 (Feature Importance Measure):
   > heads (Chefer et al., 2021, p. 786).

 * Page 77 (Outlook (0.5 p=67.5 p)):
   > Holden, C. W., & Jacobsen, S. (2014). Liquidity Measurement Problems in Fast, Competitive Markets: Expensive and Cheap Solutions: Liquidity Measure

 * Page 78 (Outlook (0.5 p=67.5 p)):
   > ment Problems in Fast, Competitive Markets. The Journal of Finance, 69(4). https://doi.org/10.1111/jofi.12127

 * Page 78 (Outlook (0.5 p=67.5 p)): "The Journal of Finance, 69(4)."

 * Page 79 (Outlook (0.5 p=67.5 p)):
   > Narang, S., Chung, H. W., Tay, Y., Fedus, W., Fevry, T., Matena, M., Malkan, K., Fiedel, N., Shazeer, N., Lan, Z., Zhou, Y., Li, W., Ding, N., Marcus,

 * Page 80 (Outlook (0.5 p=67.5 p)):
   > J., Roberts, A., & Raffel, C. (2021). Do transformer modifications transfer across implementations and applications?

## Detailed comments

 * Page 1: Anmerkung: Die gesamte Kommasetzung eventuell noch von  Fiverr Lektor korrekturlesen lassen

 * Page 3 (Trade Classification in Option Markets): Kann das irgendwie noch nach oben

 * Page 7 (Quote Rule): Eventuell andere / unterschiedlich Satzanfänge

 * Page 9 (Depth Rule): Könnte man als "wie auch immer" lesen, mit den zwei Kommas. Wäre etwas unschön.

 * Page 10 (Hybrid Rules): Der Satz möchte gerne zu seinen Freunden

 * Page 16 (Framing as a Supervised Learning Problem): For every each I get a beer on the beach

 * Page 17 (Selection of Approaches (2 p)): Absicht?

 * Page 19 (Decision Tree): Kann das eventuell noch nach oben?

 * Page 24 (Architectural Overview):

   

   Hä? Ich kenne den Drunken Sailor, aber nicht das Drunken Paper. 

   Geht hier auch eine andere Dartstellung?

 * Page 30 (Attention Mechanism):

   

   Sehr 

   langer Satz

   eventuell

 * Page 31 (Attention Mechanism): Mehr Abstand eventuell?

 * Page 32 (Attention Mechanism): Mehr Abstand eventuell?

 * Page 34 (Residual Connections and Layer Normalization):

   

   Hört sich 

   komisch an

   für mich. 

   Warum soll

   das Intuitiv 

   so sein?

 * Page 36 (FT-Transformer): Wo sind meine Kumpels? 😢

 * Page 44 (Data Collection): Letztes Zugriffsdatum eventuell angeben

 * Page 59 (Hyperparameter Tuning (2 p)): Eventuell letztes Zugriffsdatum

 * Page 61 (Evaluation Metric):

   

   Warum soll

   das intuitiv 

   sein?

 * Page 63 (Feature Importance Measure): Eventuell Abstand?

 * Page 64 (Feature Importance Measure): Eventuell Abstand?
